R/0_imports.R

```diff
@@ -26,6 +26,10 @@ tibble::type_sum
 #' @export
 tibble::size_sum
 
+#' @importFrom tibble obj_sum
+#' @export
+tibble::obj_sum
```

Comment:
Do you need to re-export `obj_sum()` and `size_sum()`? Or just import it?

Reply:
Nope. Good catch